services/QuillLMS/app/controllers/api/v1/progress_reports_controller.rb

```diff
@@ -47,15 +47,15 @@ def district_concept_reports
   def district_standards_reports
```

Comment:
I probably would have just defined a different controller action and worker here, instead of having to introduce conditional logic into both of the existing ones. I don't think it's necessary to refactor it, but just something to consider since it feels like it's really doing two totally different things. You could still have the different action/worker call the `district_standards_reports` with the freemium parameter.